stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -2080,7 +2088,9 @@ public class ChatClient internal constructor(
                 module.userStateService,
                 scope = module.networkScope,
                 retryPolicy = retryPolicy,
-            )
+            ).also {
+                configureInitializer(it)
```

Comment:
We can get rid of this once we remove ChatDomain, right?

Reply:
Now this listens for `addUserConnectedListener`. Listen for user connection is necessary because it is an asynchronous event.